stelvio/aws/function/config.py

```diff
@@ -216,26 +216,80 @@ def _validate_url(self) -> None:
 
     @property
     def folder_path(self) -> str | None:
+        """Returns the folder containing the handler code.
+
+        For "functions/orders::handler.process" → "functions/orders"
+        For "handler.process" with folder="functions/orders" → "functions/orders"
+        For "functions/users.process" without folder → None
+        """
         return self.folder or (self.handler.split("::")[0] if "::" in self.handler else None)
 
+    @property
+    def full_handler_path(self) -> str:
+        """Returns the full handler path within the project including function name.
+
+        For "functions/orders::handler.process" → "functions/orders/handler.process"
+        For "handler.process" with folder="functions/orders" → "functions/orders/handler.process"
+        For "functions/users.process" without folder → "functions/users.process"
+        """
+        if self.folder_path:
+            return f"{self.folder_path}/{self._handler_part}"
+        return self._handler_part
+
+    @property
+    def full_handler_python_path(self) -> str:
+        """
+        Docstring for full_handler_python_path
+
+        :param self: Description
+        :return: Description
+        :rtype: str
+        """
```

Comment:
This is just placeholder. Needs real docstring. 


```suggestion
        """Returns the full path to the handler Python file.

        For "functions/orders::handler.process" → "functions/orders/handler.py"
        For "handler.process" with folder="functions/orders" → "functions/orders/handler.py"
        For "functions/users.process" without folder → "functions/users.py"
        """
```